### ISSUE TITLE
[0.87.0] Realm should detect latest version

### ISSUE THREAD

**Issue by fernandodev** [realm/realm-java#1966]
It's not a major bug, but ...

I'm using Realm 0.87.0 and every each compilation I got this message: (since 0.86.1)
Note: Version 0.86.0 of Realm is now available: http://static.realm.io/downloads/java/latest

**Comment by emanuelez** [comment]
Thank you for pointing that out. It is now fixed.

**Comment by MGaetan89** [comment]
I have the same error with version 0.87.2 saying that version 0.87.0 is available.

**Comment by cmelchior** [comment]
Thanks for reporting this. We will look into it.

**Comment by cmelchior** [comment]
I could reproduce this with  `0.87.3` as well:

```
dependencies {
    compile 'io.realm:realm-android:0.87.3'
}

Version 0.87.0 of Realm is now available: http://static.realm.io/downloads/java/latest
```

**Comment by MGaetan89** [comment]
Is it really necessary to have this feature included in Realm?

**Comment by cmelchior** [comment]
This should now be fixed, also we released 0.87.4
@MGaetan89 You can see the reasoning for including it here: https://github.com/realm/realm-java/blob/master/realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmAnalytics.java

**Comment by MGaetan89** [comment]
Thanks for the information.
